queue/log.go

```diff
@@ -0,0 +1,18 @@
+package queue
+
+import log "github.com/sirupsen/logrus"
+
+// LogError is a wrapper used to log an error message
+func (qm *QueueManager) LogError(err error, message string) {
```

Comment:
Might be cool to provide an expanded interface like the one I wrote for the API error logger 😋 or even better, combine the two under a common logging utility subclass:

https://github.com/RTradeLtd/Temporal/blob/V2/api/log.go#L13

Reply:
Good idea! I will put a ticket in jira for extracting the logger functionality into a utility subclass.